Xamarin.Forms.Core/Editor.cs

```diff
@@ -1,7 +1,13 @@
 using System;
 using System.ComponentModel;
+using Xamarin.Forms;
 using Xamarin.Forms.Internals;
 using Xamarin.Forms.Platform;
+using Xamarin.Forms.StyleSheets;
+
+[assembly: StyleProperty("-xf-editor-text-color", typeof(Editor), nameof(Editor.TextColorProperty))]
+[assembly: StyleProperty("-xf-editor-placeholder", typeof(Editor), nameof(Editor.PlaceholderProperty))]
```

Comment:
we do not expose (yet) properties for setting the content (like `text`). I'd wait on this one.

Reply:
No problem. Only added this because it was in the spec. Did I miss read the spec or is the spec ahead of it's time? 

Technically, this works today, no? I mean, I specify some placeholder text and it'd be applied right? What feature is missing?

Reply:
@kingces95 I think he means we can't yet set `Text` or `Placeholder` via CSS. When he says `content`, he's referring to the CSS `content` property. That'll be a complex one to map.

Reply:
`-xf-placeholder` if it works